<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-29T15:29:56 IST" name="Suite" finished-at="2024-09-29T15:30:47 IST" duration-ms="50244">
    <groups>
    </groups>
    <test started-at="2024-09-29T15:29:56 IST" name="Test" finished-at="2024-09-29T15:30:47 IST" duration-ms="50244">
      <class name="Appium_test.TC2_eCommerce">
        <test-method is-config="true" signature="configureAppium()[pri:0, instance:Appium_test.TC2_eCommerce@55fe41ea]" started-at="2024-09-29T15:30:30 IST" name="configureAppium" finished-at="2024-09-29T15:30:35 IST" duration-ms="5475" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureAppium -->
        <test-method signature="FillForm_ErrorValidation()[pri:0, instance:Appium_test.TC2_eCommerce@55fe41ea]" started-at="2024-09-29T15:30:35 IST" name="FillForm_ErrorValidation" finished-at="2024-09-29T15:30:46 IST" duration-ms="10756" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Please enter your name> but was:<Please enter your>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Please enter your name> but was:<Please enter your>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at Appium_test.TC2_eCommerce.FillForm_ErrorValidation(TC2_eCommerce.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FillForm_ErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Appium_test.TC2_eCommerce@55fe41ea]" started-at="2024-09-29T15:30:46 IST" name="tearDown" finished-at="2024-09-29T15:30:47 IST" duration-ms="593" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Appium_test.TC2_eCommerce -->
      <class name="Appium_test.TC1_eCommerce">
        <test-method is-config="true" signature="configureAppium()[pri:0, instance:Appium_test.TC1_eCommerce@40005471]" started-at="2024-09-29T15:29:56 IST" name="configureAppium" finished-at="2024-09-29T15:30:02 IST" duration-ms="5987" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureAppium -->
        <test-method signature="FillForm(java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:Appium_test.TC1_eCommerce@40005471]" started-at="2024-09-29T15:30:03 IST" name="FillForm" data-provider="getData" finished-at="2024-09-29T15:30:24 IST" duration-ms="21123" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Srilatha M]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[female]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Argentina]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FillForm -->
        <test-method signature="ProductAddingToCart()[pri:2, instance:Appium_test.TC1_eCommerce@40005471]" started-at="2024-09-29T15:30:24 IST" name="ProductAddingToCart" finished-at="2024-09-29T15:30:30 IST" depends-on-methods="Appium_test.TC1_eCommerce.FillForm" duration-ms="5376" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductAddingToCart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Appium_test.TC1_eCommerce@40005471]" started-at="2024-09-29T15:30:30 IST" name="tearDown" finished-at="2024-09-29T15:30:30 IST" duration-ms="348" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Appium_test.TC1_eCommerce -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
